.travis/announce.sh

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env bash
+# Announce a deploy
+
+set -e
+
+function message() {
+    echo
+    echo -----------------------------------
+    echo "$@"
+    echo -----------------------------------
+    echo
+}
+
+if ! sentry-cli -V &> /dev/null
+then
+  curl -sL https://sentry.io/get-cli/ | bash
+fi
+
+ENVIRONMENT=$1
+if [ "$ENVIRONMENT" == "prod" ]; then
+TAG=latest
+elif [ "$ENVIRONMENT" == "staging" ]; then
+TAG=staging
+else
+   message UNKNOWN ENVIRONMENT
```

Comment:
Asked about this on another PR at some point but I'm wondering if we might want to `exit 1` here? I don't think we could hit this line without an error in `.travis.yml` but at the same time it feels like it could get confusing if we did happen to make that kind of error

Reply:
I actually thought there was some esoteric reason you all didn't exit here that I didn't know :)